conf/messages.en

```diff
@@ -226,14 +226,14 @@ website.title = Enter a website address
 website.heading = Enter a website address
 website.checkYourAnswersLabel = Website address
 website.error.required = Enter a website address
-website.error.length = Website address must be 100 characters or less
-website.error.duplicate = You have already added this website address
-website.error.invalid = Enter a valid website address
+website.error.length = Website address must be 250 characters or less
+website.error.duplicate = You entered this website address already. Enter a different one.
+website.error.invalid = Website address must only include letters a to z, numbers, hyphens and spaces
```

Comment:
This seems quite restrictive - we need to allow far more characters than this (e.g. `:`, `/`, `?`, `%` etc.)